gtfs/spec/en/reference.md

```diff
@@ -293,19 +294,53 @@ Primary key (`trip_id`, `stop_sequence`)
 - Simultaneous overlap of locations.geojson `id` geometry, `start/end_pickup_drop_off_window` time, and `pickup_type` or `drop_off_type` between two or more stop_times.txt records with the same `trip_id` is forbidden. For examples that demonstrate what is forbidden, see [the data example page](https://gtfs.org/schedule/examples/flex/#zone-overlap-constraint).
 
+### boarding_permissions.txt
+
+File: **Optional**
+
+Primary key (`boarding_permission_id`, `vehicle`, `start_time`)
+
+The [boarding_permissions.txt](#boarding_permissionstxt) table defines which vehicles may board, alight or be carried on a service.
+
+A vehicle can only be carried if the `pickup_permission` allows it at the origin stop,
+the `carriage_permission` allows it for every stop from the boarding stop until the stop before the alighting stop,
+and the `drop_off_permission` allows it at the alighting stop.
+
+If specified for a vehicle, if overrides the `pickup_type` and `drop_off_type` in `stop_times.txt` for passengers with that vehicle.
+To specify a vehicle is required for a passenger to use a service, set `pickup_type` and `drop_off_type` to be `1` in `stop_times.txt`,
+and set `boarding_permission_id` to reference an entry for the required vehicle in `boarding_permissions.txt`.
+
+| Field Name                | 	Type      | Presence	               | Description                                                                                                                                                                                                                                                                                                 |
+|---------------------------|------------|-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `boarding_permission_id`	 | Unique ID	 | Required	               | Identifies a boarding_permission.                                                                                                                                                                                                                                                                           |
+| `vehicle`                 | Enum       | Required	               | Identifies a vehicle. `0` - Wheelchair. `1` - Bike. `2` - Motorcycle. `3` - Car.                                                                                                                                                                                                                            |
```

Comment:
question: is it worth distinguishing between "Bike" and "Folding Bike"? For @mbta, general bikes are restricted, but folding bikes (when folded) are always allowed: https://www.mbta.com/bikes

Reply:
Yes, I will add a clarification for this such that the value only applies to full sized bikes. I don't want to define folded bikes for now as they are commonly considered luggage.